src/constants/translations/en/categories.json

```diff
@@ -0,0 +1,19 @@
+{
+        "Computer science": "Computer science",
```

Comment:
Consider using lower camel case in JSON keys.
E.g. `computerScience` instead of `Computer science`.